sql-cli/tests/test___main__.py

```diff
@@ -15,3 +15,8 @@ def test_version():
     result = runner.invoke(app, ["version"])
     assert result.exit_code == 0
     assert "Astro SQL CLI 0.0.0" in result.stdout
+
+
+def test_validate():
+    result = runner.invoke(app, ["validate"])
+    assert result.exit_code == 0
```

Comment:
It would be great if we could test other aspects of the feature, e.g.:
- did it create the connection?
- did it log the expected message?
- did it not log unwanted messages?

We may want to have separate tests to cover:
- if the connection existed with a different value, was it replaced? It would be nice to not print (deleted, valid, created in those cases - only replaced)
- if the connection existed with the same value, could we just report to the end-user that it is valid?
- if the connection is not valid

Reply:
That looks very nice, thanks, @pankajkoti ! We can have a separate ticket to add colours if things pass or fail.